### PR TITLE
Collapsible Input Categories

### PR DESCRIPTION
**Pull request by TheRealFalseReality**
I updated the blueprint with collapsible inputs, maybe it'll make it a litter easier to configure by hiding some options.

### CODE REVIEW

**Comment by TheRealFalseReality** [comment]
Screenshot 2024-08-19 013130

**Comment by SgtBatten** [comment]
Oh nice find, thank you.

File: Frigate Camera Notifications/Stable [hunk 1]
```diff
@@ -115,639 +115,667 @@ blueprint:
...
+      name: |
         # Notification Customisations
+      description: These are Notification Customisation Options
+      icon: mdi:bell
+      collapsed: true
+      input:
+        title:
+          name: Notification Title (Optional)
+          description: |
+            # Notification Customisations
```
Comment (SgtBatten) [inline Frigate Camera Notifications/Stable:128]:
```suggestion
          description: The title of the notification.
```

File: Frigate Camera Notifications/Stable [hunk 2]
```diff
@@ -115,639 +115,667 @@ blueprint:
...
 
-        The title of the notification.
-      default: ""
-    message:
-      name: Notification Message
-      description: |
-        The message of the notification.
-        You can use variables such as {{camera_name}} and {{label}}
-        e.g., A {{ label }} {{ 'is loitering' if loitering else 'was detected' }} on the {{ camera_name }} camera.
+            The title of the notification.
```
Comment (SgtBatten) [inline Frigate Camera Notifications/Stable:130]:
```suggestion
```

File: Frigate Camera Notifications/Stable [hunk 3]
```diff
@@ -115,639 +115,667 @@ blueprint:
...
+          description: |
+            When enabled, the notification will stay active on the device after tapping it and remain there until cleared.
+          default: false
+          selector:
+            boolean:
+        channel:
+          name: Notification Channel - Android only (Optional)
+          description: Create a new channel for notifications to allow custom notification sounds, vibration patterns, and override of Do Not Disturb mode. Configured directly on the device.
+          default: ""
+    filers:
```
Comment (SgtBatten) [inline Frigate Camera Notifications/Stable:341]:
```suggestion
    filters:
```

**Review by SgtBatten — CHANGES_REQUESTED** [review]
Also please change the version to 0.12.0.4b

**Comment by TheRealFalseReality** [comment]
> 0.12.0.4b

yea it was introduced in 2024.6

**Pull request merged**